docs/src/components/ComponentDoc/ComponentDocAccessibility.tsx

```diff
@@ -21,17 +29,89 @@ function getStem(info) {
 }
 
 function getBehaviorName(stem) {
-  const filename = stem && `${stem}.ts`
+  if (!stem) {
+    return undefined
+  }
+  const filename = stem && `${stem}.ts`.replace(stem[0], stem.charAt(0).toLowerCase())
+  for (let i = 0; i < behaviorMenu.length; i++) {
+    const behaviour = behaviorMenu[i].variations.find(
+      behaviorVariation => behaviorVariation.name === filename,
+    )
+    if (behaviour) {
+      return behaviorMenu[i].displayName
+    }
+  }
+}
+
+function getId(idName: string) {
+  return _.kebabCase(idName)
+}
+
+function getAvailableBehaviorsFromJson(availableBehaviors: any): any[] {
+  const availableBehaviorsFromJson = []
+  availableBehaviors.map(availableBehavior => {
+    behaviorMenu.map(behaviorInJson => {
+      const result = behaviorInJson.variations.find(
+        variation => variation.name === `${availableBehavior.name}.ts`,
+      )
+      if (result) {
+        availableBehaviorsFromJson.push(result)
+      }
+      return
```

Comment:
unnecessary returns?

Reply:
Juraj proposed change will change this part of code and "return" will be not needed...